.hashibot.hcl

```diff
@@ -0,0 +1,12 @@
+poll "closed_issue_locker" "locker" {
+    schedule = "0 10 5 * * *"
+    closed_for = "720h" # 30 days
+    no_comment_if_no_activity_for = "360h" # 15 days
```

Comment:
I just realized that since you are locking another that's been closed for 30 days, that means likely EVERY issue being locked hasn't had activity for 15 days, meaning none of these issues will have comments. I would recommend setting the value for `no_comment_if_no_activity_for` to maybe 60 days `1440h`?

Reply:
Oh that makes sense - I misunderstood the behaviour of that